distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc

```diff
@@ -12,15 +12,15 @@
 |security.access-groups
 |Attribute name for storing groups to enforce access
 controls upon
-|String
-|
+|List of Strings
+|< 1024 characters per entry
 |
 
 |security.access-individuals
 |Attribute name for storing the email addresses of users to
 enforce access controls upon
-|String
-|Valid email address
+|List of Strings
+|< 1024 characters per entry; Valid email address
```

Comment:
Not sure if these email ones need the character constraint here. Looks like emails can be 256 characters at the longest http://www.rfc-editor.org/errata_search.php?rfc=3696

Also should it be `Valid email address per RFC-3696`?